### PR TITLE
Feat: Add Missing Documentation for Mobile Destination Plugins [Netlify Build]

### PR DESCRIPTION
**Pull request by alanjcharles**
### Proposed changes

- Added missing documentation for mobile plugins 

- new documentation for `Adobe Analytics`,  `Localytics`, `Intercom`, and `Nielsen`. 
- fixed a typo in the SDK index files. 

### Merge timing
- Once it is proof-read and approved by my team. 
- I can follow up with here or with Paul Wooly directly for more accurate timing info once I have it.

### CODE REVIEW

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/images/nielsen.png
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/mobile/kotlin-android/images/nielsen.png:1]: should this appear in the docs for either of the kotlin nielsen plugins?

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/images/projectstructure.png
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/mobile/kotlin-android/images/projectstructure.png:1]: does this need to appear in the docs for the Nielsen DCR plugin?

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+
+## Adding the dependency
+
+***Note:** the Intercom library itself will be installed as an additional dependency.*
+
+### via Xcode
+In the Xcode `File` menu, click `Add Packages`.  You'll see a dialog where you can search for Swift packages.  In the search field, enter the URL to this repo.
+
+https://github.com/segment-integrations/analytics-swift-intercom
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md:29]: yep totally agree

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+The bundled mobile integrations also check `properties.total` if `properties.revenue` is not present, and assign the total value as the amount value.
+
+### Limited Properties
+Intercom can only store [five event properties](http://docs.intercom.io/Intercom-for-user-analysis/Tracking-User-Events-in-Intercom#metadata-support){:target="_blank"} per event. That means if you send an event to Segment with more than five properties, Intercom only shows the first five properties.
+
+### Limited Events
+
+Intercom only allows a total of 120 unique _active_ event names. If you're sending Segment more than 120 unique event names, Intercom only accepts the first 120 events that their servers see, and the rest throw an error.
+
+In Intercom, an "Active" event is an event that hasn't been archived. If you archive an event, it makes it inactive and removes it from your 120 active events. If you need to bring your account back under the 120 event limit, archive some events from in the Intercom UI by navigating to **Settings > (workspace name) data > Events**, then click on the event to archive.
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md]: I don't off the top of my head. This is all copied from our existing Intercom doc

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+// ...
+
+analytics.group(groupId: "group123", traits: MyTraits(
+        username: "MisterWhiskers",
+        email: "hello@test.com",
+        plan: "premium"))
+```
+
+Segment supports Intercom companies in all sources. Users can be put into multiple groups, which associate them to multiple companies in Intercom.
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md:196]: it supports sending Intercom's concept of `Company` from all Segment source types.

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/localytics-swift.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+When you call [`identify`](/docs/connections/spec/identify/), we'll set the Localytics
+customer Id, and set any special Localytics traits you provide, such as `name`,
+or `email`, and any custom traits as well.
+
+## Track
+
+Whenever you call [`track`](/docs/connections/spec/track/), we'll log an event with
+Localytics. [`track`](/docs/connections/spec/track/) takes the name of the event and any
+optional properties you want to associate with the event.
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/localytics-swift.md]: no they could be whatever the customer wants to add to their track calls- from something in our spec to a totally custom value. All we're trying to say here is that they can send anything as a property and Intercom will accept it/we don't map it to anything specific.

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/nielsen-dcr.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Nielsen Digital Content Ratings (DCR) respond to the shifting, complex multi-platform, multi-device and multi-distribution landscape by providing comprehensive measurement of digital content consumption—including streaming video, static web pages and mobile apps—across all major devices and platforms. The [Analytics-Swift Nielsen-DCR Plugin](https://github.com/segment-integrations/analytics-swift-nielsen-dcr) tracks sessions via for [Analytics-Swift](https://github.com/segmentio/analytics-swift).
+
+## Getting Started
+
+In order to get started with Nielsen-DCR and retrieve an appid to configure this integration, you must sign a license agreement on the Nielsen engineering portal.
+
+There will be an NDA to sign prior to accessing the download. Nielsen requires you fill out your company info and have a Nielsen representative before getting started.
+
+You must also go through a pre-certification process with your Nielsen representative before shipping this implementation to production.
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/nielsen-dcr.md]: Genuinely not sure. Again, this is info copied from our existing public-facing doc so I am assuming the general Segment set up will be the same

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/nielsen-dcr.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
+title: Analytics Swift Nielsen DCR Plugin
+strat: swift
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/nielsen-dcr.md]: ah, good to know- thanks!

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/adobe-kotlin-android.md [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+## Sending Events
+
+You can map Segment events in your `Events V2` settings to any event variable you already defined in your Adobe Analytics Mobile Services dashboard.
+
+> warning ""
+> **Note**: Do not use the deprecated `Events` settings. These no longer forward events to Adobe.
+
+Here's an example of how you might map Segment events to Adobe Analytics events connected in device mode:
+
+<!--todo: rewrite this so that it doesn't rely on the screenshots to explain how to map events in Segment-->
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/adobe-kotlin-android.md:82]: hmmm this just accidentally got pasted from the existing docs, i think! I would be down to help but I don't believe we have access to Adobe (we contracted this out). We should also figure out what to do with this in the existing docs

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/intercom-kotlin-android.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+  intercom.logout()
+```
+
+## Best Practices
+
+### Arrays and Objects
+
+Intercom doesn't support custom arrays or objects. This means that if you want to send a certain user `trait` or event `property` to Intercom, you must send them at the top level.
+
+This limitation does not apply, however, for mapping `company` objects on [Identify calls](/docs/connections/spec/identify/). Segment continues to handle that in the same way as before. This is only applicable for any custom traits or properties.
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/intercom-kotlin-android.md]: yep! (again copied from existing docs

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dcr-kotlin-android.md [hunk 9]
```diff
@@ -0,0 +1,246 @@
+---
+title: Analytics Kotlin Nielsen DCR Plugin
+strat: kotlin
+---
+
+Nielsen Digital Content Ratings (DCR) respond to the shifting, complex multi-platform, multi-device and multi-distribution landscape by providing comprehensive measurement of digital content consumption—including streaming video, static web pages and mobile apps—across all major devices and platforms. The [Analytics-Kotlin Nielsen-DCR Plugin](https://github.com/segment-integrations/analytics-kotlin-nielsen-dcr) tracks data for [Analytics-Kotlin](https://github.com/segmentio/analytics-kotlin).
+
+## Getting Started
+
+### Adding the dependency
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dcr-kotlin-android.md:16]: oops- good catch!

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dcr-kotlin-android.md [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+
+The SDK is started by calling the play API with the `channelName` and `mediaURL` parameters. If no value is passed in, Segment defaults to `defaultChannelName` for `channelName` and an empty String for `mediaURL`.
+
+#### Cross Id 1
+
+Standard episode ID.
+
+#### Cross Id 2
+
+Content originator ID. This value is only required for distributors.
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dcr-kotlin-android.md:236]: unclear what the change is here?

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dtvr-kotlin-android.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
+---
+title: Analytics Kotlin Nielsen DTVR Plugin
+strat: kotlin
+---
+
+Nielsen Digital Content Ratings (DCR) respond to the shifting, complex multi-platform, multi-device and multi-distribution landscape by providing comprehensive measurement of digital content consumption—including streaming video, static web pages and mobile apps—across all major devices and platforms. The [Analytics-Kotlin DTVR Plugin](https://github.com/segment-integrations/analytics-kotlin-nielsen-dtvr) tracks data for [Analytics-Kotlin](https://github.com/segmentio/analytics-kotlin).
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dtvr-kotlin-android.md]: oops- good catch

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dtvr-kotlin-android.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+store, after receiving Nielsen certification.
+
+These new settings will take up to an hour to propagate to all of your existing
+users. For new users it will be instantaneous.
+
+#### Enable Debug Mode 
+Check this setting if you would like to activate the
+Debug flag. Once the flag is active, it logs each API call made and the data
+passed. DO NOT activate the Debug flag in a production environment.
+
```
[the remark this answers is not in the document]
Comment (alanjcharles) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dtvr-kotlin-android.md:99]: not sure off the top of my head  (copied from existing public-facing doc)

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/adobe-swift.md [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+## Sending Events
+
+You can map Segment events in your `Events V2` settings to any event variable you already defined in your Adobe Analytics Mobile Services dashboard.
+
+> warning ""
+> **Note**: Do not use the deprecated `Events` settings. These no longer forward events to Adobe.
+
+Here's an example of how you might map Segment events to Adobe Analytics events connected in device mode:
+
+<!--todo: rewrite this so that it doesn't rely on the screenshots to explain how to map events in Segment-->
```
[the remark this answers is not in the document]
Comment (forstisabella) [reply src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/adobe-swift.md:86]: if it was copied from the original doc, I'll add a backlog item to fix this but it shouldn't be a blocker!

File: src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md [hunk 14]
```diff
@@ -0,0 +1,231 @@
...
```
Comment (forstisabella) [inline src/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift.md:196]:
```suggestion
Segment supports Intercom `company` values sent from all source types. Users can be put into multiple groups, which associate them to multiple companies in Intercom.
```

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dcr-kotlin-android.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (forstisabella) [reply src/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/nielsen-dcr-kotlin-android.md:236]: I think if we put context about each option above the code snippet, the code snippet would be more clear!

**Review by forstisabella — APPROVED** [review]

**Comment by alanjcharles** [comment]
Hi @forstisabella! I think we're all good here so feel free to make any additional changes you think are necessary and release whenever it's convenient.

**Comment by forstisabella** [comment]
@alanjcharles sweet! I'll release it this morning with our regular docs release cadence. Thanks for the great work you did on this!

**Pull request merged**